### ISSUE TITLE
ft_checkdata fails on segmentation without unit

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/fieldtrip#1459]
**Describe the bug**

With #1451 and the change https://github.com/fieldtrip/fieldtrip/commit/539145e6bee7c87f7af164245722ce1440323d05#diff-80f320a88d5e4e533150eb9c82a65448 @schoffelen introduced extra feedback on each of the segmentations. 

However, it assumes that segmentation.unit is present, which is not always the case. In most high-level FT code it will be added by the hasunit option in `ft_checkdata`, but that will only have effect *after* the call to ft_datatype_segmentation.

**To Reproduce**

```
>> ft_checkdata(rmfield(segmentedmri, 'unit'))
Warning: more than 4 tissue types, this may take a while 
Reference to non-existent field 'unit'.
Error in ft_datatype_segmentation (line 301)
  u = sprintf('%s^3', segmentation.unit);
Error in ft_checkdata (line 276)
  data = ft_datatype_segmentation(data, 'segmentationstyle', segmentationstyle, 'hasbrain', hasbrain); 
```
**Expected behavior**

It should either estimate the units, or print the dimensions without units.

**Screenshots**

This is FieldTrip, revision 9d2c88264.

**Comment by robertoostenveld** [comment]
Looking at the code, I see that all other feedback is handled by ft_checkdata and not by ft_datatype_xxx. I will change that as well.

**Comment by robertoostenveld** [comment]
After doing some initial reogganization, nother issue is the following

```
>> ft_checkdata(segmentedmri, 'feedback', 'yes')
the input is segmented volume data with dimensions [256 256 256]
Reference to non-existent field 'inside'.
Error in ft_checkdata>print_segmentedvolumes (line 1839)
  tmp = repmat(segmentation.inside, [1 1 1 numel(fn)]);
Error in ft_checkdata (line 207)
      print_segmentedvolumes(data) 
```

I discussed this already with @schoffelen and will use the probabilistic values as fractional volumes, e.g., a voxel that is 50% grep matter and 50% white matter will contribute 0.5 mm^3 to both volumes.

**Comment by robertoostenveld** [comment]
fixed with the merge of #1460
